src/pages/homeCollection/assigned.js

```diff
@@ -13,10 +13,10 @@ export const assignedScreen = async (auth, route) => {
 
 const assignedTemplate = async (name, auth, route) => {
   showAnimation();
-  const response = await getParticipantSelection("assigned");
+  const response = await getParticipantsByKitStatus("assigned");
```

Comment:
May need error handling.

Reply:
This file is part of the kit status reports that is disabled and still a work in progress (old code that needs to be changed).